spec/open_weather/endpoints/stations_spec.rb

```diff
@@ -94,4 +94,95 @@
       end
     end
   end
+
+  describe '#create_measurements' do
+    it 'creates measurements', vcr: { cassette_name: 'stations/create_measurement_success' } do
+      create_params = {
+        "station_id": '5ed21a12cca8ce0001f1aef1',
+        "dt": 1479817340,
+        "temperature": 18.7,
+        "wind_speed": 1.2,
+        "wind_gust": 3.4,
+        "pressure": 1021,
+        "humidity": 87,
+        "rain_1h": 2,
+        "clouds": [
+          {
+            "condition": 'NSC'
+          }
+        ]
+      }
+      data = client.create_measurements([create_params])
+      expect(data).to be_nil
+    end
+
+    context 'when station does not exist' do
+      it 'raises error', vcr: { cassette_name: 'stations/create_measurement_failed_with_invalid_station' } do
+        create_params = {
+          "station_id": -1,
+          "dt": 1479817340,
+          "temperature": 18.7,
+          "wind_speed": 1.2,
+          "wind_gust": 3.4,
+          "pressure": 1021,
+          "humidity": 87,
+          "rain_1h": 2,
+          "clouds": [
+            {
+              "condition": 'NSC'
+            }
+          ]
+        }
+        expect { client.create_measurements([create_params]) }
+          .to raise_error(OpenWeather::Errors::Fault, /expected=string, got=number/)
+      end
+    end
+  end
+
+  describe '#get_measurements' do
+    it 'gets measurements', vcr: { cassette_name: 'stations/get_measurement_success' } do
+      data = client.get_measurements(
+        station_id: '5ed21a12cca8ce0001f1aef1',
+        type: 'd',
+        limit: 100,
+        from: 1469817340,
+        to: 1591620047
+      )
+      expect(data.size).to eq(1)
+      expect(data.first).to be_a(OpenWeather::Models::Stations::Measurement)
```

Comment:
```ruby
station = data.first
expect(station.x)....
```

Reply:
heh i thought typing `data.first` was shorter than typing `measurement`, will change it though